_typos.toml

```
@@ -0,0 +1,13 @@
+# Files for typos
+# Instruction:  https://github.com/marketplace/actions/typos-action#getting-started
+
+[default.extend-identifiers]
+
+[default.extend-words]
+# Don't correct following words
+nin="nin"
+nd="np" # nd may be np (numpy)
```

Look like this is what was meant in those docstrings:
```suggestion
nd.array="np.ndarray"
```

---

I don't understand how this works, from the documentation it looks like it's used to add words to the dictionary (to ignore them), or is it to force corrections? Or both? How was the `nd.array` typo detected in the first place, was it Typo that pointed it out or did @shirayu saw it themselves and so decided to add it here?

---

I tried typo detection of `np.ndarra` with this change but it did not work.
My understanding is that typos first tokenize the file and then check using the dictionary. 
Therefore, the words that contain `. ` are not recognized well.